src/components/Connection/Connection.spec.tsx

```diff
@@ -15,8 +16,8 @@ function renderConnection(props: Partial<ConnectionProps>) {
       i18n={{
         title: 'Unlock Your Virtual World.',
         subtitle: 'Access and start exploring.',
-        accessWith: option => `Access with ${option}`,
-        connectWith: option => `Connect with ${option}`,
+        accessWith: option => `Access with ${option.toString().replace('-', '')}`,
+        connectWith: option => `Connect with ${option.toString().replace('-', '')}`,
```

Comment:
I didn't like this solution. You can move it to a function and give a logic like:

```
function getWalletNameFromOption(option: String) {
  if (option === "wallet-connect") {
    return "Wallet Connect"
  }
  return option
}
```

and then apply it to the option like `getWalletNameFromOption(option)`

Reply:
done

Reply:
added const ConnectionOptionTitle